### ISSUE TITLE
Erase not all setting by "Erase all settings from board"

### ISSUE THREAD

**Issue by Zyamstein** [TauLabs/TauLabs#1648]
Mine situation
board - CC3D
TL - next's and master 10/26/14
not erasing autotune computed values

**Comment by peabody124** [comment]
Did you exit GCS and reconnect?

**Comment by Zyamstein** [comment]
yep,ad make it about 10 times and on few next build's and master build from 10/26/14
can make a video or share pc table to show it
p.s. - I use windows version
